### DISCUSSION TITLE
PNG to SVG

### DISCUSSION THREAD

**Discussion by 010sfs** [brend8c/FFMPEG_BAT#6] [Q&A] [answered]
I think this is a great idea!
How to convert png to svg using ffmpeg?

**Comment by brend8c** [comment, marked as answer]
@010sfs,
Ffmpeg can't convert raster graphics to vector!

**Comment by brend8c** [comment]
@010sfs, You can use this library
https://www.npmjs.com/package/svg-png-converter
